### PR TITLE
Adds buttons to print multiple items at protolathe

### PR DESCRIPTION
**Pull request by AnturK**
Makes the protolathe menu look even more cluttered but printing components for machine upgrades is bit easier now. Could probably make buttons show only when you got materials for all of them but then i would have to add some cache so it's not checking all that crap every single time  or just turn the whole thing into a queue but i'm not that motivated.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
pull from ntstation master

it will fix the compile error

**Comment by GunHog** [comment]
Dis is guud :+1:

**Comment by Iamgoofball** [comment]
will do soon ish

**Comment by AnturK** [comment]
Probably need to add resize of the window to this since the default one is bit too narrow

**Comment by Atskadan** [comment]
needs bugtesting

**Comment by Atskadan** [comment]
sad that this isnt bugtested by today

**Comment by Atskadan** [comment]
@anturk please

**Comment by AnturK** [comment]
Uh, not sure what you're asking for. I can update the branch if that's what you mean.

**Comment by Iamgoofball** [comment]
we gotta have someone bugtest it

Hold tight, I'll look into it.

**Comment by Atskadan** [comment]
@Iamgoofball did you test it?

**Comment by GunHog** [comment]
I am testing this now. The machine spams "not enough materials" for every item in the queue. The queue should be cleared once any of the items fail to be produced.

**Comment by Atskadan** [comment]
@anturk please fix ^

**Comment by AnturK** [comment]
Sorry for the delay, had pretty crazy schedule. Should be good now, also widened the console a bit.

**Comment by ghost** [comment]
merge when

**Comment by Iamgoofball** [comment]
someone merge this

File: code/modules/research/rdconsole.dm [hunk 1]
```diff
@@ -141,6 +141,93 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 	griefProtection()
 */
 
+/obj/machinery/computer/rdconsole/proc/build(var/id,var/amount)
+	//TODO
```
Comment (RemieRichards) [inline code/modules/research/rdconsole.dm:145]: explain further

**Comment by RemieRichards** [comment]
@Iamgoofball Not until the "TODO" appearing out of nowhere is explained.

**Comment by Atskadan** [comment]
well was it explained

**Pull request merged**